ext_mongodb.php

```diff
@@ -746,7 +868,7 @@ public function getCode() : string
 
 	public function getScope() : mixed
 	{
-		if ( $this->scope !== NULL )
+		if ( isset( $this->scope) && $this->scope !== NULL )
```

Comment:
If scope is defined as a property, it shows up in __set_state through var_export. I have no control over that. I can't decided whether to include it or not, it's how HHVM works. So if the solution is that you change get_properties to always include scope as well, that's fine. Otherwise, my current code should stay.